sansio_lsp_client/client.py

```diff
@@ -211,7 +410,7 @@ def shutdown(self) -> None:
 
     def exit(self) -> None:
         assert self._state == ClientState.SHUTDOWN
-        self._send_notification(method="exit")
+        self._send_notification(method="exit", params={})
```

Comment:
I'd like it if we could put a comment here explaining why we can't just not specify `params`

Reply:
I thought this was another gopls thing, but changing it to `params=None` doesn't break gopls tests.

Reply:
I added a TODO comment about this

Reply:
It was indeed added in gopls fixes, but I don't remember the events leading up to it, and I already deleted all the go stuff, sorry.